packages/lesswrong/lib/vulcan-lib/utils.ts

```diff
@@ -78,7 +78,8 @@ export const getSiteUrl = function (): string {
   if (url.slice(-1) !== '/') {
     url += '/';
   }
-  return url;
+  // return url;
```

Comment:
Probably better to change the siteUrlSetting (the `siteUrl` instance setting) and leave a comment in the script itself that it needs to be changed to avoid sending out urls with localhost in them